src/Symfony/Component/Notifier/Bridge/Novu/NovuSubscriberRecipient.php

```diff
@@ -26,6 +26,7 @@ public function __construct(
         private readonly string|null $phone = null,
         private readonly string|null $avatar = null,
         private readonly string|null $locale = null,
+        private readonly array $overrides = [],
```

Comment:
Would be good to have at least a basic phpdoc with definition of what this array can contain